decidim-core/app/packs/src/decidim/index.js

```diff
@@ -104,7 +104,7 @@ $(() => {
   })
   document.querySelectorAll(".new_report").forEach((container) => changeReportFormBehavior(container))
 
-  updateExternalDomainLinks($("body"))
+  document.querySelectorAll("a").forEach((elem) => updateExternalDomainLinks(elem))
```

Comment:
After #10967 will be merged, this line needs to be checked. In index.js we already have a snippet like: 

```
  document.querySelectorAll("a[target=\"_blank\"]:not([data-external-domain-link=\"false\"])").forEach((elem) => new ExternalDomainLink(elem))
```

Also, this line is different from the one in redesigned_index.js

Reply:
It's different due to the backwards compatibility, since I changed the `updateExternalDomainLinks` function input arguments.
In the redesigned_index, what I did was merging both snippets into a single one, that is, if some link doesn't contain `data-external-link=false`, then it applies the external-domain-warning also. I believe both functions are closely related, and perhaps we can save on a `data-attribute`,

Thoughts? cc/ @andreslucena

Reply:
I know what you mean, as I also have the doubt while working on #10121.

As long as it works as it should (ie we should not show the external domain warning if a domain is whitelisted), it sounds good to me to reuse the same selector logic.